adam-cli/src/main/scala/org/bdgenomics/adam/cli/Transform.scala

```diff
@@ -121,14 +130,11 @@ class Transform(protected val args: TransformArgs) extends BDGSparkCommand[Trans
     if (args.recalibrateBaseQualities) {
       log.info("Recalibrating base qualities")
       val knownSnps: SnpTable = createKnownSnpsTable(sc)
-      val stringency = if (args.strictBQSR) {
```

Comment:
Your question makes sense. I guess the short answer is that yes, I think the two notions of stringency are "independent" and unlikely to both need to be specified (let alone to different values) in the same `transform` run.

I'd go a step further though and say that it's worth formalizing the idea that, in the context of any one `transform` command, there might be a level of strictness of sanity/validation-checking that we should expose to the user via a cmdline flag. On the other hand, though, exposing multiple flags related to this basic idea feels redundant / like over-doing it, to me.